ci/Jenkinsfile

```
@@ -9,181 +9,183 @@ def wait_for_db() {
     }
 }
 
-// REQUIRED for Deployment
-// see Jenkinsfile.release
-stage('Build Schema'){
-    node('docker') {
-        node_info()
+timeout(time: 1, unit: 'MINUTES') {
+    disableConcurrentBuilds()
```

Otherwise it would simply wait?

```suggestion
    disableConcurrentBuilds(abortPrevious: true)
```

---

Yea I was not aware how to do this, had to check the jenkins doc.

---

Doesn't seem to work eitherway :/